src/Router.php

```diff
@@ -277,18 +274,22 @@ public function clearAll(): void
     protected function invokeRoute(
         ServerRequestInterface $request,
         ResponseInterface $response,
-        RouteRunner $runner,
+        ?RouteRunner $runner = null,
```

Comment:
You can drop `?` in function signature for `null` default values.

Reply:
This is needed for throwing 404